lhotse/kaldi.py

```diff
@@ -97,8 +107,10 @@ def fix_id(t: str) -> str:
                 )
             ],
             sampling_rate=sampling_rate,
-            num_samples=compute_num_samples(durations[recording_id], sampling_rate),
-            duration=durations[recording_id],
+            num_samples=compute_num_samples(
+                float(durations[recording_id]), sampling_rate
```

Comment:
I think it could be cleaner to do sth like `load_kaldi_text_mapping(path, float_vals: bool = False)` and when `float_vals=True`, cast the second column to float in the function. Otherwise you have to cast in multiple places, looks like a nasty surprise for anybody modifying these things in the future :)